BlueprintUI/Sources/Layout/Builder/EqualStack+Builder.swift

```diff
@@ -0,0 +1,15 @@
+extension EqualStack {
```

Comment:
I would just add these inits in `EqualStack.swift` (and same for the other files) – no reason to split it out, and it makes the API harder to find if you're just browsing that file otherwise.